skypeweb/skypeweb_login.c

```diff
@@ -167,10 +169,12 @@ skypeweb_login_got_t(PurpleUtilFetchUrlData *url_data, gpointer user_data, const
 			gchar *session_state = skypeweb_string_get_chunk(url_text, len, ":'https://login.live.com/GetSessionState.srf?", "',");
 			if (session_state) {
 				//These two appear to have different object keys each request :(
+				/*

```

Comment:
This is work in progress code for the two factor auth implementation, showing how it should be done, if it's ever done.